enos/enos-scenario-e2e-docker.hcl

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# For this scenario to work, add the following line to /etc/hosts
+# 127.0.0.1 localhost boundary
+
+scenario "e2e_docker" {
+  terraform_cli = terraform_cli.default
+  terraform     = terraform.default
+  providers = [
+    provider.aws.default,
+    provider.enos.default
+  ]
+
+  locals {
+    aws_ssh_private_key_path = abspath(var.aws_ssh_private_key_path)
+    local_boundary_dir       = abspath(var.local_boundary_dir)
+    tags = merge({
+      "Project Name" : var.project_name
+      "Project" : "Enos",
+      "Environment" : "ci"
+    }, var.tags)
+  }
+
+  step "create_docker_network" {
+    module = module.docker_network
+  }
+
+  step "create_boundary_database" {
+    depends_on = [
+      step.create_docker_network
+    ]
+    variables {
+      network_name = step.create_docker_network.network_name
+    }
+    module = module.docker_postgres
+  }
+
+  step "create_boundary" {
+    module = module.docker_boundary
+    depends_on = [
+      step.create_docker_network,
+      step.create_boundary_database
+    ]
+    variables {
+      image_name       = var.boundary_docker_image
+      network_name     = step.create_docker_network.network_name
+      postgres_address = step.create_boundary_database.address
+    }
+  }
+
+  step "create_vault" {
+    module = module.docker_vault
+    depends_on = [
+      step.create_docker_network
+    ]
+    variables {
+      image_name   = "docker.io/hashicorp/vault:1.12.2"
```

Comment:
Did we not version Vault somewhere centrally already? Or is this maybe a reason to introduce such a central place?

Reply:
Yup we did 😅